pandas/core/arrays/string_.py

```diff
@@ -282,8 +284,25 @@ def astype(self, dtype, copy=True):
         return super().astype(dtype, copy)
 
     def _reduce(self, name, skipna=True, **kwargs):
+        if name in ["min", "max"]:
+            return getattr(self, name)(skipna=skipna, **kwargs)
```

Comment:
You don't need to pass kwargs here, as there are no possible additional ones for min/max as far as I am aware